.github/workflows/push-asset-readme-update.yml

```diff
@@ -0,0 +1,16 @@
+name: Plugin asset/readme update
+on:
+  push:
+    branches:
+    - master
+jobs:
+  master:
+    name: Push to master
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: WordPress.org plugin asset/readme update
```

Comment:
We don't commit `pot` file so we need to generate it before deploying to svn.

Reply:
Note that our plugin on wp.org doesn't have a localization file atm.

Reply:
@dinhtungdu I added a POT file generation step to our release process, want to pick that up on your Monday and then progress through any remaining steps on this release?

Reply:
Should we include it in our workflow? Eg:
```
 - uses: actions/checkout@master
    - name: Build
      run: |
        npm install
        ./node_modules/.bin/grunt
        ./node_modules/.bin/grunt i18n
```

Reply:
I've just found that w.org runs `wp i18n make-pot` automatically so we don't need to generate pot file here, resolved.